container-search-gui/src/main/resources/gui/_includes/search-api-reference.html

```diff
@@ -0,0 +1,1011 @@
+---
```

Comment:
This is a maintenance problem - why do you want to have a copy?

Reply:
I just did not know how to read from the actual file from the documentation-repo..
Also, I needed a few changes in the parameters id, for getting information on all parameters (eg. collapsesize --> collapse.size)